Source/WebCore/xml/XMLHttpRequest.cpp

```diff
@@ -682,7 +681,7 @@ void XMLHttpRequest::abort()
 {
     Ref<XMLHttpRequest> protectedThis(*this);
 
-    m_wasAbortedByClient = true;
+    m_wasAbortedByClient = false;
```

Comment:
The web page is calling abort() so it seems `m_wasAbortedByClient` should be true.
If we change the meaning of `m_wasAbortedByClient`, we should change its name.
In this case, maybe we can remove it altogether.

Reply:
You're right, not sure what happened here. I'll restore.